dev/TabView/TabView.cpp

```diff
@@ -44,13 +54,48 @@ void TabView::OnApplyTemplate()
         }
         return scrollViewer;
     }());
+
+    m_addButton.set([this, controlProtected]() {
+        auto addButton = GetTemplateChildT<winrt::Button>(L"AddButton", controlProtected);
+        if (addButton)
+        {
+            // Do localization for the add button
+            auto addButtonName = ResourceAccessor::GetLocalizedStringResource(SR_TabViewAddButtonName);
+            winrt::AutomationProperties::SetName(addButton, addButtonName);
+
+            m_addButtonClickRevoker = addButton.Click(winrt::auto_revoke, { this, &TabView::OnAddButtonClick });
+        }
+        return addButton;
+    }());
+}
+
+winrt::IVector<winrt::FrameworkElement> TabView::GetAccessibleChildElements()
```

Comment:
>GetAccessibleChildElements [](start = 49, length = 26)

What calls this?

Reply:
Oh thank you, no one does -- that was left over from my abandoned accessibility changes. Removed.